recipe/meta.yaml

```diff
@@ -26,6 +29,7 @@ requirements:
     - pip
   run:
     - python
+    - {{ pin_compatible("numpy") }}
```

Comment:
```suggestion
```

This isn't actually needed anymore given information provided in https://github.com/conda-forge/mt2-feedstock/pull/1